src/backend/InvenTree/report/tasks.py

```diff
@@ -68,7 +68,7 @@ def print_labels(
     model = template.get_model()
     items = model.objects.filter(pk__in=item_ids)
 
-    plugin = registry.get_plugin(plugin_slug)
+    plugin = registry.get_plugin(plugin_slug, active=True)
```

Comment:
Comment: would it make sense to filter to active plugins by default? Makes unintended use less likely